client/src/commands.ts

```diff
@@ -23,30 +23,39 @@ interface Command {
  * @param client language client
  * @param context extension context for adding disposables
  */
-function restartNgServer(client: lsp.LanguageClient, context: vscode.ExtensionContext): Command {
+function restartNgServer(client: AngularLanguageClient): Command {
   return {
     id: 'angular.restartNgServer',
     async execute() {
       await client.stop();
-      context.subscriptions.push(client.start());
+      await client.start();
     },
   };
 }
 
 /**
  * Open the current server log file in a new editor.
  */
-function openLogFile(client: lsp.LanguageClient): Command {
+function openLogFile(client: AngularLanguageClient): Command {
   return {
     id: 'angular.openLogFile',
     async execute() {
       const serverOptions: ServerOptions|undefined = client.initializeResult?.serverOptions;
       if (!serverOptions?.logFile) {
-        // TODO: We could show a MessageItem to help users automatically update
-        // the configuration option then restart the server, but we currently do
-        // not reload the server options when restarting the server.
-        vscode.window.showErrorMessage(
-            `Angular Server logging is off. Please set 'angular.log' and reload the window.`);
+        // Show a MessageItem to help users automatically update the
+        // configuration option then restart the server.
+        const selection = await vscode.window.showErrorMessage(
+            `Angular server logging is off. Please set 'angular.log' and restart the server.`,
+            'Enable logging and restart Angular server',
```

Comment:
```suggestion
        const selection = await vscode.window.showErrorMessage(
            `Angular server logging is off. Please set 'angular.log' and restart the server.`,
            'Enable verbose logging and restart Angular server',
            `Set 'angular.log' manually (server will not be restarted)`,
```

Reply:
Actually, we don't need to add the second option, because if users want to set the verbosity manually, they could just ignore this message or hit `escape`.